mappings/net/minecraft/client/gui/screen/GraphicsConfirmationScreen.mapping

```diff
@@ -0,0 +1,14 @@
+CLASS net/minecraft/class_5405 net/minecraft/client/gui/screen/GraphicsConfirmationScreen
+	FIELD field_25675 message Lnet/minecraft/class_5348;
+	FIELD field_25676 choiceButtons Lcom/google/common/collect/ImmutableList;
```

Comment:
```suggestion
	FIELD field_25676 buttons Lcom/google/common/collect/ImmutableList;
```

Reply:
wouldn't this clashes with `Screen`'s `buttons` protected member?

Reply:
Ah yes, only suggested that to match the param